x-pack/plugins/security_solution/public/management/pages/policy/view/protection_updates/protection_updates_layout.tsx

```diff
@@ -358,6 +360,11 @@ export const ProtectionUpdatesLayout = React.memo<ProtectionUpdatesLayoutProps>(
 
     return (
       <>
+        <ProtectionUpdatesWarningPanel
```

Comment:
Should we display it only when switching to not automatic updates?

Also, what about adding the condition of displaying it or not here instead of inside the component? Something like:
```typescript
return (
  <>
    { showWarningPanel && <ProtectionUpdatesWarningPanel ...
...
```

So no empty component needs to be rendered.

Reply:
@caitlinbetz Based on mockups I assumed we want it shown for both active an inactive automatic updates, is this assumption correct?